src/Cluster/coverage.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#$ -V
+#$ -N sequdas_coverage
+#$ -cwd
+#$ -pe smp 25
```

Comment:
Is it necessary to request 25 slots for this job?

Reply:
Right, I forgot to change all of the files.

Thanks Dan!